movieHWTests/MockRouter.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MockRouter.swift
+//  movieHWTests
+//
+//  Created by Александр Троицкий on 17.01.2023.
+//
+
+import XCTest
+
+@testable import movieHW
```

Comment:
```suggestion
@testable import movieHW
```
почему movieHW с маленькой?